cloudify_deployment_proxy/__init__.py

```diff
@@ -304,13 +311,23 @@ def post_execute_node_instance_proxy(self):
         return True
 
     def post_execute_deployment_proxy(self):
+        runtime_prop = ctx.instance.runtime_properties['deployment']
+        ctx.logger.debug(
+            'Runtime  deployment properties {}'.format(runtime_prop))
 
         if 'outputs' \
                 not in ctx.instance.runtime_properties['deployment'].keys():
             update_attributes('deployment', 'outputs', dict())
+            ctx.logger.debug('No deployment proxy outputs exist !!!!')
```

Comment:
Can we do with a `.` instead of ` !!!!` :)

Reply:
done